dd-sdk-android/src/main/kotlin/com/datadog/android/tracing/AndroidTracer.kt

```diff
@@ -193,6 +194,15 @@ class AndroidTracer internal constructor(
             return this
         }
 
+        /**
+         * Sets the sampling rate of spans.
+         * @param samplingRate the sampling rate as a percentage between 0 and 100 (default is 100%)
+         */
+        fun setSamplingRate(samplingRate: Double): Builder {
```

Comment:
we can also add `FloatRange` annotation for the clarity.

Reply:
Makes sense indeed